### DISCUSSION TITLE
Installazione HA Core su Docker

### DISCUSSION THREAD

**Discussion by vitovitale93** [indomus/forum#3324] [HUB: Home Assistant]
Ciao a tutti, sto provando a installare HA tramite Raspberry 4 con Docker Compose. Sto seguendo la guida sul sito ma ho dei problemi con la creazione del container
<img width="1040" alt="Screenshot 2024-12-21 alle 18 29 40" src="https://github.com/user-attachments/assets/5f04f29d-71e6-4b4c-bd86-46c29e75b30f" />
Sono arrivato a questo punto e non so come andare avanti, credo di dover creare un container per Home Assistant con il seguente comando:

nano home-assistant.yaml

e copiarci dentro quello che mi dice la guida, giusto? Scusate ma mi sto affacciando solo ora a queste cose

**Comment by snakuzzo** [comment]
Quel comando non è che ti fa creare il container, ma editare il file che ti servirà per creare poi il/i container.
Però perdonami, prima di scrivere qui, hai provato a leggere tutto tentando di comprendere? Ho l'impressione che tu stia seguendo la guida facendo solo copia-incolla, se vai avanti e continui a leggere c'è scritto tutto.

**Comment by vitovitale93** [comment]
Si ho letto tutto e provato in vari modi cercando di guardare anche altrove, ma ho sempre lo stesso problema. Ora ho creato il file di HA con il comando  

nano home-assistant.yaml

dove inserisco la parte evidenziata in blu della guida, sotto al blocco services, e una volta salvata provo a farla partire ma mi restituisce questo errore 
<img width="428" alt="Screenshot 2024-12-22 alle 00 04 41" src="https://github.com/user-attachments/assets/2e74b022-e3bb-4ba7-b916-c9f780a983d6" />

in questo punto la guida mi risulta un po confusionaria portandoti da una guida all'altra, ma più o meno qualcosa devo averla capita, almeno credo

**Reply by snakuzzo** [reply]
Stai sbagliando tutto.
Rileggi la guida da capo, per favore, e capirai subito dove hai sbagliato

**Reply by snakuzzo** [reply]
Senza andare troppo nel dettaglio, perché certe cose te le puoi andare a leggere sulla doc ufficiale di docker

O chiami il file `docker-compose.yaml` e dai semplicemente il comando `docker compose up -d` 

Oppure usi il comando `docker compose -f home-assistant.yaml up -d`

**Reply by snakuzzo** [reply]
Consiglio...passa a HA OS. Non è ancora per te docker...e non ti si può seguire passo passo per ogni cosa del genere

**Reply by vitovitale93** [reply]
Mi sono letto un pò il doc ufficiale di Docker e credo di aver capito l'errore che facevo, adesso rifaccio un pò di prove e vediamo. Per ora ho tempo per sbatterci la testa quindi continuo a provare con compose. Mi spieghi solo che cambia utilizzando il comando Docker run?
<img width="726" alt="Screenshot 2024-12-22 alle 10 30 51" src="https://github.com/user-attachments/assets/7dee657f-de05-4ffb-b2f1-6d3adf953dfd" />

**Reply by vitovitale93** [reply]
<img width="921" alt="Screenshot 2024-12-22 alle 10 36 48" src="https://github.com/user-attachments/assets/327ee7d4-b6d3-44ac-99a4-dbe283318a8f" />
sembra che ci siamo
